media-video/ffmpeg/ffmpeg-4.0.2.recipe

```diff
@@ -0,0 +1,304 @@
+	lib:libavfilter$secondaryArchSuffix = $libavfilterVersionCompat
+	lib:libavformat$secondaryArchSuffix = $libavformatVersionCompat
+	lib:libavresample$secondaryArchSuffix = $libavresampleVersionCompat
+	lib:libavutil$secondaryArchSuffix = $libavutilVersionCompat
+	lib:libpostproc$secondaryArchSuffix = $libpostprocVersionCompat
+	lib:libswresample$secondaryArchSuffix = $libswresampleVersionCompat
+	lib:libswscale$secondaryArchSuffix = $libswscaleVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libmodplug$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libspeex$secondaryArchSuffix
+	lib:libtheoradec$secondaryArchSuffix
+	lib:libtheoraenc$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libvorbisenc$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	REQUIRES+="
+		lib:libvpx$secondaryArchSuffix
+		lib:libgnutls$secondaryArchSuffix
+		"
+fi
+CONFLICTS="
+	ffmpeg_legacy$secondaryArchSuffix
+	ffmpeg${secondaryArchSuffix}_bin
+	ffmpeg2${secondaryArchSuffix}_bin
+	"
+
+PROVIDES_avdevice="
+	ffmpeg${secondaryArchSuffix}_avdevice = $portVersionCompat
+	lib:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	"
+REQUIRES_avdevice="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix == $libavcodecVersion
+	lib:libavfilter$secondaryArchSuffix == $libavfilterVersion
+	lib:libavformat$secondaryArchSuffix == $libavformatVersion
+	lib:libavutil$secondaryArchSuffix == $libavutilVersion
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	ffmpeg${secondaryArchSuffix}_tools = $portVersion
+	cmd:ffmpeg = $portVersionCompat
+	cmd:ffplay = $portVersionCompat
+	cmd:ffprobe = $portVersionCompat
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix == $libavcodecVersion
+	lib:libavdevice$secondaryArchSuffix == $libavdeviceVersion
+	lib:libavfilter$secondaryArchSuffix == $libavfilterVersion
+	lib:libavformat$secondaryArchSuffix == $libavformatVersion
+	lib:libavutil$secondaryArchSuffix == $libavutilVersion
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	ffmpeg${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libavcodec$secondaryArchSuffix = $libavcodecVersionCompat
+	devel:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	devel:libavfilter$secondaryArchSuffix = $libavfilterVersionCompat
+	devel:libavformat$secondaryArchSuffix = $libavformatVersionCompat
+	devel:libavresample$secondaryArchSuffix = $libavresampleVersionCompat
+	devel:libavutil$secondaryArchSuffix = $libavutilVersionCompat
+	devel:libpostproc$secondaryArchSuffix = $libpostprocVersionCompat
+	devel:libswresample$secondaryArchSuffix = $libswresampleVersionCompat
+	devel:libswscale$secondaryArchSuffix = $libswscaleVersionCompat
+	"
+REQUIRES_devel="
+	ffmpeg$secondaryArchSuffix == $portVersion base
+	ffmpeg${secondaryArchSuffix}_avdevice == $portVersion
```

Comment:
The devel package contains symlinks to the libavdevice, so it needs to be added here.
Review required.

Reply:
Seems fine to me